### ISSUE TITLE
CGU et mentions legales

### ISSUE THREAD

**Issue by Luthaf** [patacrep/patanet#74]
On a déjà un embryon de mentions légales (`/fr/pages/about`), mais il faudrait les améliorer, et ajouter les CGU et la politique de confidentialité. Au passage, devons-nous déclarer le site a la CNIL ?

A faire : 
- [ ] Mentions légales
  - [ ] Hébergeur
  - [ ] Mentions CNIL
- [ ] CGU
  - [ ] Licence des chants/carnet
  - [ ] Utilisation du mail pour les informations uniquement
  - [ ] Politique de confidentialité

Pour les CGU, quelque chose comme [ca](http://progdupeu.pl/pages/cgu) peut être pas mal.
Qu'ai-je oublie ?

Les deux peuvent être regroupées dans une seule page.

**Comment by Soundy25** [comment]
Oui, il faut déclarer le site à la CNIL à cause du service d'enregistrement des données personnelles (nom+prenom+mail) des utilisateurs et puis j'imagine que vous ouvrirez également la Newsletter des contributions du mois, donc autant le faire directement, non ? 
A mon avis, il ne s'agit que d'une déclaration (pas d'une demande d'autorisation), aucune donnée sensible (santé, religion, sécurité, origines) n'est traitée par ce service. 
Je peux me rencarder là-dessus.

**Comment by Soundy25** [comment]
http://www.cnil.fr/vos-obligations/declarer-a-la-cnil/dispense/mon-secteur-dactivite/mon-theme/mon-fichier/dispense-selectionnee/dec-mode/DISPLAYSINGLEFICHEDISP/dis-uid/38/?tx_oxcscnildeclaration_pi1[sauid]=0&tx_oxcscnildeclaration_pi1[tuid]=0&cHash=c3a74d11e8a0a37723657455eb47d703 

Résumé

La dispense n°6 concerne les sites web ou blogs mis en œuvre par des particuliers à titre privé qui peuvent permettre, d’une part, la collecte de données à caractère personnel de personnes qui s’y connectent et, d’autre part, la diffusion de données à caractère personnel (nom, images de personnes ou tout autre élément permettant d’identifier une personne physique).

La diffusion et la collecte de données à caractère personnel opérées à partir d’un site web dans le cadre d’activités professionnelles, politiques, ou associatives restent soumises à une déclaration préalable auprès de la CNIL.

http://www.cnil.fr/vos-obligations/declarer-a-la-cnil/dispense/mon-secteur-dactivite/mon-theme/mon-fichier/dispense-selectionnee/dec-mode/DISPLAYSINGLEFICHEDISP/dis-uid/45/?tx_oxcscnildeclaration_pi1[sauid]=0&tx_oxcscnildeclaration_pi1[tuid]=0&cHash=9915950e909feedafbe68f54ff4d8684
Résumé

La dispense de déclaration n°8 (ancienne norme simplifiée n°23) concerne les traitements de données personnelles mis en oeuvre par tout organisme à but non lucratif (association loi 1901, fondations, fonds de dotation) pour la gestion administrative de leurs membres, bénévoles et donateurs. Elle concerne également les annuaires des membres y compris ceux diffusés sur internet ainsi que toute action de prospection réalisée auprés membres, donateurs.

La dispense prévoit que seules peuvent être enregistrées les données relatives à l’identité, l’identité bancaire, vie associative, et à des fins statistiques les données de connexion. Elle exclut les données sensibles telles que les origines raciales ou ethniques, les opinions politiques, philosophiques ou religieuses, l’appartenance syndicale, l’état de santé ou la vie sexuelle des personnes, les infractions, condamnations ou mesure de justice, les informations sur les difficultés sociales et le numéro de sécurité sociale.

Ces données ne peuvent être conservées au delà de la démission ou de la radiation du membre (sauf s’il fait la demande contraire) et pour les donateurs au delà de deux sollicitations restées infructueuses. Les informations sur les prospects ne sont pas conservées après la réalisation de la campagne de prospection Les personnes concernées doivent être informées lors de la collecte des données de toute opération visant à diffuser leurs données personnelles, ainsi que sur leur droit d'opposition, d'accès et de rectification et sur les modalités d'exercice de ces droits. Leur consentement doit être obtenu si l’association envisage de céder ou louer leurs coordonnées à des fins de prospection commerciale par voie electronique (e-mailing).

**Comment by Soundy25** [comment]
Exemple de mention CNIL à afficher sur le formulaire d’inscription :

Les informations recueillies sont nécessaires pour votre adhésion.
Elles font l’objet d’un traitement informatique et sont destinées à la gestion des accès du site et à la diffusion de la newsletter. En application des articles 39 et suivants de la loi du 6 janvier 1978 modifiée, vous bénéficiez d’un droit d’accès et de rectification aux informations qui vous concernent.

Si vous souhaitez exercer ce droit et obtenir communication des informations vous concernant, veuillez vous adresser à : [indiquez-ici le service chargé du droit d'accès].

**Comment by Soundy25** [comment]
Il faudrait aussi guetter ça d'un peu plus près :
http://www.cnil.fr/vos-obligations/sites-web-cookies-et-autres-traceurs/

**Comment by oliverpool** [comment]
A propos de la légalité de patanet : http://en.wikipedia.org/wiki/On-line_Guitar_Archive (cas découvert via http://tenbyten.com/software/songsgen/) et la suite : http://arstechnica.com/tech-policy/2007/04/legal-guitar-tabs-return-to-the-web/ (musicnotes est désormais payant d'après ce que j'ai vu)
